src/sentry/static/sentry/app/components/group/sentryAppExternalIssueForm.tsx

```diff
@@ -11,30 +11,46 @@ import {t} from 'app/locale';
 import ExternalIssueStore from 'app/stores/externalIssueStore';
 import getStacktraceBody from 'app/utils/getStacktraceBody';
 import withApi from 'app/utils/withApi';
-
-class SentryAppExternalIssueForm extends React.Component {
-  static propTypes = {
+import {Client} from 'app/api';
+import {Group, PlatformExternalIssue, Event, SentryAppInstallation} from 'app/types';
+import {Field} from 'app/views/settings/components/forms/type';
+
+type Props = {
+  api: Client;
+  group: Group;
+  sentryAppInstallation: SentryAppInstallation;
+  appName: string;
+  config: object;
+  action: 'create' | 'link';
+  event: Event;
+  onSubmitSuccess: (externalIssue: PlatformExternalIssue) => void;
+};
+
+//TODO(TS): Improve typings on Field an input so we can use the type in functions without errors
```

Comment:
Not sure what this comment is referring to here

Reply:
@billyvg In many functions, `field` is typed as `any` instead of `Field`. This is because I keep getting typing errors. Examples:

![Screen Shot 2019-10-23 at 8 53 31 AM](https://user-images.githubusercontent.com/8533851/67411583-b566c980-f572-11e9-833e-a69a3b00f804.png)
![Screen Shot 2019-10-23 at 8 54 15 AM](https://user-images.githubusercontent.com/8533851/67411584-b566c980-f572-11e9-9256-9044c116e5ab.png)